launcher.js

```diff
@@ -63,6 +66,9 @@ const is_garbage = n => {
         // xlink -> <use -> "S"> <use -> "ponsored">
         return true
     }
+    if (n.textContent.startsWith(lang["Suggested"]) || n.textContent.startsWith(lang["Suggested"].substr(1))) {
+        return true
```

Comment:
Have you encountered the need of `.substr(1)`? If not, put it rather below to the block:
```
if (!n.children.length) {
        if ([lang["Suggested for you"],
        lang["Suggested live gaming broadcast"],
        lang["Suggested groups"],
        lang["People You May Know"],
        lang["Friend Requests"],
        lang["Videos Just For You"]]
```

Reply:
No. I'll update it